examples/address/create_adress.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'easypost'
+
+EasyPost.api_key = ENV['EASYPOST_TEST_API_KEY']
```

Comment:
Issue: Let's replace all occurrences of `EASYPOST_TEST_API_KEY` with `EASYPOST_API_KEY` to be more generic since people can use either/or for many of these.